### PR TITLE
Add a workflow to automatically bump versions on main

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

Depends on https://github.com/tektoncd/operator/pull/932

/cc @sm43 @concaf @tektoncd/operator-maintainers 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Run `make test lint` before submitting a PR
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by vdemeester** [comment]
Next step is to do the same on release branches, but using `bump-bugfix` command instead 👼🏼

File: .github/workflows/bump-payload-on-main.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+on:
+  schedule:
+  # Run this every week day at 1AM
+  - cron: '0 1 * * 1-5'
+
+jobs:
+  bump-payloads:
+    name: "Bump payloads"
+    runs-one: ubuntu-latest
```
Comment (PuneetPunamiya) [inline .github/workflows/bump-payload-on-main.yaml:11]: Should it be `runs-on` ?

Comment (vdemeester) [reply .github/workflows/bump-payload-on-main.yaml:11]: indeed 😝

**Comment by sm43** [comment]
Build Error
```
 ./.github/workflows/bump-payload-on-releases.yaml
  3:1       warning  truthy value should be one of [false, true]  (truthy)
  25:19     error    syntax error: mapping values are not allowed here (syntax)

```

**Comment by vdemeester** [comment]
Fixed yamls 👼🏼

File: .github/workflows/bump-payload-on-releases.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        echo '::set-output name=branches::[${RELEASE_BRANCHES}]'
+    outputs:
+      branches: ${{ steps.set-matrix.outputs.branches }}
+  bump-payloads:
+    needs: build-release-matrix
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        branch: ${{ fromJSON(needs.build-matrix.outputs.branches) }}
+    - uses: actions/setup-go@v2
```
Comment (piyush-garg) [inline .github/workflows/bump-payload-on-releases.yaml:25]: `steps:` is missing

Comment (vdemeester) [reply .github/workflows/bump-payload-on-releases.yaml:26]: fixed 👼🏼

**Pull request merged**